### PR TITLE
Ifucube radial weighting and new reference files 

### PR DESCRIPTION
**Pull request by jemorrison**
DO NOT MERGE UNTIL NEW CUBEPARS REFERENCE FILES IN CRDS.
The PR addresses #3761 
Updated cube_build to read in the new reference files that contain the information to set up the radial weighting scheme.

### CODE REVIEW

File: jwst/cube_build/instrument_defaults.py [hunk 1]
```diff
@@ -605,42 +683,50 @@ def GetSoftRad(self, parameter1, parameter2):
...
         return table
 
     def Get_high_table(self):
         table = (self.high_wavelength,
                  self.high_sroi,
                  self.high_wroi,
                  self.high_power,
-                 self.high_softrad)
+                 self.high_softrad,
+                 self.hight_scalerad)
```
Comment (philhodge) [inline jwst/cube_build/instrument_defaults.py]: Shouldn't this be `high_scalerad`?

**Comment by drlaw1558** [comment]
Some quick comments from testing things:
- Is there no check that the specified weighting function exists in the cubepar file?  I.e., if I call cube_build with weighting="aardvark" I just get ERROR RUNNING STEP 'CubeBuildStep':
    Floating point nan values are not allowed in FITS headers
after a while.  This crash will be what we see until cubepar gets updated in crds.

- Overriding the reference file with my local new cubepar file and weighting="emsm" I get a data cube that doesn't look like I'd expect, details over on Jira ticket.

**Comment by drlaw1558** [comment]
It looks like @jemorrison latest commit fixed the weighting issue where the radial scale radius wasn't being converted to pixel units properly.  I'm going to do some more quick tests with the updated code, updated the new cubepar file and deliver it to crds.  Once it's active in CRDS I'll approve this PR.

**Comment by jdavies-st** [comment]
Restarted TravisCI build now that new cubepars reffiles are in CRDS.

File: jwst/cube_build/instrument_defaults.py (hunk 1 above)
Comment (jemorrison) [reply jwst/cube_build/instrument_defaults.py]: yes - fixed

**Comment by jemorrison** [comment]
@drlaw1558 
@hbushouse 
@stscieisenhamer 
@jdavies-st 
This PR is ready to merge. The reference files are in CRDS. This pr will result in cube_build using a different weighting function to combine data, so the regression tests will fail. Since we are in the middle of updating how the regression tests are being done I don't know if we should hold off on this one or merge it ?

**Comment by drlaw1558** [comment]
Where are we on being able to merge this PR?  Since this is a blocker issue for other work it would be helpful to merge this asap since all of the reference files are ready.

**Comment by jemorrison** [comment]
it is ready to be merged. It will change the output cube_build files. @hbushouse @stscieisenhamer  the regression tests will fail. Do you want me to update the truth files before we merge this ?
We are still running the old regression system nightly - correct ?

**Comment by hbushouse** [comment]
Yes, we're still running both the old and new regression systems, at least for now. We're hoping to turn off the old ones soon. Go ahead and merge and then look at the differences that result, then update the truth files.

**Pull request merged**